docs/source/installed_packages.rst

```diff
@@ -0,0 +1,114 @@
+.. _installed-packages:
+
+Using Installed Packages
+========================
+
+`PEP 561 <https://www.python.org/dev/peps/pep-0561/>`_ specifies how to mark
+a package as supporting type checking. Below is a summary of how to create
+PEP 561 compatible packages and have mypy use them in type checking.
+
+Making PEP 561 compatible packages
+**********************************
+
+Packages that must be imported at runtime and supply type information should
```

Comment:
It's not entirely clear what "packages that must be imported at runtime" means. Can you rephrase it? Does this means packages with type annotations included directly in the package implementation?